.github/workflows/ci.yaml

```diff
@@ -0,0 +1,159 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+
+      - name: Build
+        run: npm exec -- lerna run build --concurrency=2
+
+      - name: Upload dist artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist-artifact
+          path: packages/**/dist
+
+  build-demo:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+
+      - name: Build Demo
+        run: npm run build:demo
+
+      - name: Upload demo artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: demo-artifact
+          path: ./demo
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+
+      - name: Lint
+        run: npm run lint:ci
+
+  test:
+    needs: [build]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+
+      - name: Download dist artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-artifact
+          path: packages
+
+      - name: Test
+        run: npm run test:ci
+
+      - name: Coveralls
+        if: env.COVERALLS_REPO_TOKEN != ''
+        run: npm exec -- coveralls < .cache/coverage/lcov.info
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+
+  deploy:
+    needs: [build-demo, lint, test]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+
+      - name: Download demo artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: demo-artifact
+          path: ./demo
+
+      - name: Deploy
+        run: utils/scripts/deploy.mjs
+        env:
+          NETLIFY_TOKEN: ${{ secrets.NETLIFY_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+          # Maintain compatibility with @zendeskgarden/scripts
+          CIRCLE_PROJECT_USERNAME: zendeskgarden
+          CIRCLE_PROJECT_REPONAME: react-components
+          CIRCLECI: true
```

Comment:
Temporary solution to avoid errors. Ultimately, we could update `githubRepository` to support [HTTP URLs](https://github.com/zendeskgarden/react-components/actions/runs/9462583243/job/26066350814#step:6:9).

https://github.com/zendeskgarden/scripts/blob/82b001563247320e98315a30c6c4fe45db9f8950/src/github/repository/index.ts#L34-L52

Reply:
If GH actions provides standard owner & repo string in the env, then we should definitely get `scripts` updated for use with this PR.

Reply:
It does within the Github Context.

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context